echoregions/regions2d/regions2d.py

```diff
@@ -330,41 +275,30 @@ def convert_points(
         points : list or dict
             single converted point or list/dict of converted points depending on input
         """
-        return self._parser.convert_points(
-            points,
-            convert_time=convert_time,
-            convert_depth_edges=convert_depth_edges,
-            offset=offset,
-            unix=unix,
-        )
 
-    def get_points_from_region(
-        self, region_id: Union[int, str, Dict], file: str = None
-    ) -> List:
-        """Get points from specified region from a JSON or CSV file
-        or from the parsed data.
-        Parameters
-        ----------
-        region_id : int, str, or dict
-            ID of the region to extract points from or region dictionary
-        file : str
-            path to JSON or CSV file. Use parsed data if None
-        Returns
-        -------
-        points : list
-            list of x, y points
-        """
-        return self.plotter.get_points_from_region(region_id, file)
-
-    def _init_plotter(self) -> None:
-        """Initialize the object used to plot regions."""
-        if self._plotter is None:
-            if self.data is None:
-                raise ValueError(
-                    "Input file has not been parsed; call `parse_file` to parse."
-                )
+        def _swap_depth_edge(self, y: Union[int, float]) -> Union[int, float]:
```

Comment:
I am really confused by this and `convert_points` -- I don't see it used anywhere in the repo. Not sure if I am missing anything though. The use of 9999.99 and -9999.99 suggests that there's some redundancy with `replace_nan_depth`. Do we need to keep the `convert_points` function?